Dynatrace/dynatrace-operator
# Adds toleration for arm and amd to operator components

# Description

Adds tolerations to operator components so that they are deployed on ARM nodes as well

## How can this be tested?

* Acquire an ARM cluster
* Generate the manifest
* Deploy the manifest (an arm image with the snapshot-<branch> tag exists already)
* Observe


## Checklist
